cleantalk.antispam/options.php

```diff
@@ -135,6 +137,21 @@
             COption::SetOptionString( $sModuleId, 'site_exclusions', $cleantalk_antispam_default_settings['site_exclusions']);
         }
         COption::SetOptionString( $sModuleId, 'form_exclusions_url',             isset($_POST['form_exclusions_url'])     ? $_POST['form_exclusions_url']     : $cleantalk_antispam_default_settings['form_exclusions_url'] );
+
+        if (
+            isset($_POST['form_exclusions_url']) &&
+            ! empty($_POST['form_exclusions_url']) &&
+            isset($_POST['form_exclusions_url__regexp']) &&
+            ct_is_valid_regexp($_POST['form_exclusions_url'])
```

Comment:
ct_is_valid_regexp() is not defined

Reply:
Saw it in the other pull https://github.com/CleanTalk/bitrix-antispam/pull/19/commits